WebApp-your-API/MyOrg/Client/Pages/ToDoPages/Edit.razor

```diff
@@ -15,11 +16,16 @@
     ToDo toDo = new ToDo();
     protected async override Task OnParametersSetAsync()
     {
-        toDo = await ToDoListService.GetAsync(Id);
+        toDo = await _downstreamApi.GetForUserAsync<ToDo>(
+                "TodoList",
+                options => options.RelativePath = $"api/todolist/{Id}");
     }
+
     protected async Task EditTask()
     {
-        await ToDoListService.EditAsync(toDo);
+        await _downstreamApi.CallApiForUserAsync<ToDo, ToDo>(
+                 "TodoList", toDo,
+                 options => { options.RelativePath = $"api/todolist/{Id}"; options.HttpMethod = HttpMethod.Patch; });
```

Comment:
Is there any reason you didn't use PostApiForUser?

Reply:
I was following the given example too closely :). Also, this is a patch (post produces a "method not allowed" exception, so I will use PatchForUserAsync